manifests/q/Qoppa/PDFStudio/2021.0.2/Qoppa.PDFStudio.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Qoppa.PDFStudio
+PackageVersion: 2021.0.2
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- pdf
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VC++2015-2019Redist-x64
+Installers:
+- Platform:
+  - Windows.Desktop
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://download.qoppa.com/pdfstudio/PDFStudio_win64.msi
+  InstallerSha256: 0C49420DD4D5DB8C01A6CFC1D5B74D52AA7D11DF0BD572A7EA9D47934AAB9543
+  ProductCode: '{5B75DF97-B6E1-49B7-9C73-6850760E5200}'
```

Comment:
```
PS C:\Users\WDAGUtilityAccount\Desktop\manifests> winget install -m "C:\Users\WDAGUtilityAccount\Desktop\manifests\q\Qoppa\PDFStudio\2021.0.2"
Found PDF Studio [Qoppa.PDFStudio]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.qoppa.com/pdfstudio/PDFStudio_win64.msi
  ██████████████████████████████   190 MB /  190 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

```suggestion
  ProductCode: '{5B75DF97-B6E1-49B7-9C73-6850760E5200}'
  InstallerSwitches:
    Custom: /q
```

Reply:
watafak, i though the /q in the custom parameter was weird and unneeded since its a MSI, i've seen this before but its very very rare